packages/new-block-stream/README.md

```diff
@@ -4,6 +4,87 @@ A tool for creating a stream of new block data.
 
 ## Usage
 
+### Getting started
+
+Install via npm:
+
+```js
+npm install @drizzle-utils/new-block-stream
+```
+
+Usage example (taken from `@drizzle-utils/new-block-stream/example.js`)
+
 ```js
-// TODO
+/* eslint-disable no-console */
+const Web3 = require("web3");
+const createNewBlock$ = require("@drizzle-utils/new-block-stream");
+
+const main = async () => {
+  // const web3 = new Web3("http://127.0.0.1:9545"); // HttpProvider
+  const web3 = new Web3("ws://127.0.0.1:9545"); // WebsocketProvider
+
+  const { observable, subscription } = createNewBlock$({
+    web3,
+    pollingInterval: 200, // only used if non-WebsocketProvider
+  });
+
+  // log out new blocks (might skip blocks if polling)
+  const stream = observable.subscribe(console.log);
+
+  // 10 seconds later, unsubscribe
+  setTimeout(() => {
+    console.log("unsubscribing...");
+    subscription.unsubscribe();
+    stream.unsubscribe();
+    process.exit();
+  }, 10000);
+};
+
+main();
 ```
+
+## API
+
+### createNewBlock$
+
+Creates an RxJS observable that will monitor the blockchain for new blocks. Supports both [Web3 subscriptions](https://web3js.readthedocs.io/en/1.0/web3-eth-subscribe.html#eth-subscribe) and polling.
+
+#### Parameters
+
+options - `Object`
```

Comment:
Variable names should be in backticks: `options`.

Same goes for `web3`, `pollingInterval`, `observable`, `subscription`, `unsubscribe` below.